composer.json

```diff
@@ -30,7 +30,14 @@
     "authors": [
         {
             "name": "Romain Norberg",
-            "email": "romainnorberg@gmail.com"
+            "email": "romainnorberg@gmail.com",
+            "role": "Maintainer, Owner, Developer"
+        },
+        {
+            "name": "Mocking Magician",
```

Comment:
✅  I had to done this before. sorry

Reply:
Don't worry, one way or another I participate of my own free will ;-)